yt_dlp/extractor/manoto.py

```diff
@@ -0,0 +1,100 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+
+class ManotoTVIE(InfoExtractor):
+    IE_NAME = 'Manoto TV (Episode)'
+    _VALID_URL = r'https?://(?:www\.)?manototv\.com/episode/(?P<id>[0-9]+)'
+    _TEST = {
+        'url': 'https://www.manototv.com/episode/12576',
+        'info_dict': {
+            'id': '12576',
+            'title': 'Seh Mah Taatili',
+            'description': 'مجموعه ای از فیلم های سینمای کلاسیک ایران',
+            'thumbnail': r're:^https?://.*\.jpeg$',
+            'ext': 'm3u8',
+        }
+    }
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        episode_json = self._download_json('https://dak1vd5vmi7x6.cloudfront.net/api/v1/publicrole/showmodule/episodedetails?id=' + video_id, video_id)
+        details = episode_json.get('details', {})
+        title = details.get('analyticsEpisodeTitle')
+        video_url = details.get('videoM3u8Url')
+        description = details.get('showSynopsis')
+        thumbnail = details.get('episodelandscapeImgIxUrl')
+        ext = 'm3u8'
+        formats = self._extract_m3u8_formats(video_url, video_id, ext)
```

Comment:
```suggestion
        formats = self._extract_m3u8_formats(video_url, video_id, 'mp4')
```

It's probably mp4, but make sure

Reply:
Done.